index.js

```diff
@@ -80,7 +111,7 @@ function lint(source, options, webpack, done) {
         emitter = webpack.emitWarning;
       }
 
-      emitter(options.formatter(report));
+      emitter(new Error(options.formatter(report)));
```

Comment:
aha, I realised what was going wrong, travis + my local dev envrionment uses the yarn.lock which has an older version of webpack in the dependencies. It seems like webpack introduced a breaking change between `2.2.1` and `2.4.1` that throws if you emit non errors. If you sync your branch with master the tests should start passing 😄